api_core/google/api_core/bidi.py

```diff
@@ -499,6 +499,7 @@ def __init__(self, bidi_rpc, on_response):
         self._wake = threading.Condition()
         self._thread = None
         self._operational_lock = threading.Lock()
+        self._ready = threading.Event()
```

Comment:
just pass this as an argument to _thread_main so its usage is clear?

Reply:
Sure. I can do that. Since it is internal it won't break the interface.